frontend/src/styles/globals.css

```
@@ -1,4 +1,18 @@
-@import url('https://fonts.googleapis.com/css2?family=Fraunces:opsz,wght@9..144,400;9..144,500;9..144,600;9..144,700&family=Manrope:wght@400;500;600;700;800&display=swap');
+@font-face {
+  font-family: 'Fraunces';
```

Stylelint (17.2.0)

[error] Unexpected quotes around "Fraunces" (font-family-name-quotes)

(font-family-name-quotes)